src/main/java/com/autotune/analyzer/deployment/AutotuneDeployment.java

```diff
@@ -57,787 +59,806 @@
 /**
  * Maintains information about the Autotune resources deployed in the cluster
  */
-public class AutotuneDeployment
```

Comment:
It looks like there has been a space / tab change that is causing the whole of the code to show up as changed. This makes it very difficult to review the actual changes. Can we keep the original formatting, so that the actual changes get highlighted

Reply:
Done